var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -13,7 +13,7 @@ class Qgis(CMakePackage):
     """
 
     homepage = "https://qgis.org"
-    url = "https://qgis.org/downloads/qgis-3.8.1.tar.bz2"
+    url = "https://qgis.org/downloads/qgis-latest.tar.bz2"
```

Comment:
This version does not exist in the package, and will cause `spack url summary` to report incorrect metrics.

Reply:
So it is useless to merge this PR. Thank you anyway.

Reply:
Yes, the way to add new versions is to add the `version` directives with the `sha256` checksums. The URL is deduced automatically. This can be done easily with `spack checksum qgis`.